sphinx/themes/basic/static/basic.css_t

```diff
@@ -52,6 +52,7 @@ div.sphinxsidebar {
     width: {{ theme_sidebarwidth|toint }}px;
     margin-left: -100%;
     font-size: 90%;
+    word-wrap: break-word;
```

Comment:
`overflow-wrap : break-word;` also needed. I'll add it after merging.
https://developer.mozilla.org/en/docs/Web/CSS/word-wrap